cloudwatch/cloudwatch.go

```diff
@@ -165,6 +167,14 @@ func newCloudWatchLogsClient(roleARN string, sess *session.Session, endpoint str
 		}
 		svcConfig.EndpointResolver = endpoints.ResolverFunc(cwCustomResolverFn)
 	}
+	if credsEndpoint != "" {
+		logrus.Infof("Trying to get credentials from credentials_endpoint %s", credsEndpoint)
```

Comment:
This info statement is redundant since you already print it when you read the parameter.

Reply:
Yeah, makes sense. will remove it